tests/acceptance/features/bootstrap/Sharing.php

```
@@ -1682,6 +1682,19 @@ public function getLastShareIdOf($user) {
 		return $id;
 	}
 
+	/**
+	 * Retrieves the id of the last shared file
+	 *
+	 * @return string|null
+	 */
+	public function getLastShareId() {
+		if ($this->lastShareData) {
+			return $this->lastShareData->data[0]->id;
+		} else {
+			return null;
```

I think I would prefer to throw an error here, because otherwise we would not notice if no id was found

---

I don't think we can do that because this function is currently used by `substituteInlineCodes`, which checks if a function returns the value or null and returns the original if null is returned.
Furthermore, the function signature states that it returns `string|null` so the consumers should check themself if the returned value is null and handle the error accordingly.